substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfSections.java

```diff
@@ -26,6 +26,15 @@
 
 package com.oracle.objectfile.elf.dwarf;
 
+import java.nio.ByteOrder;
```

Comment:
Sorry for the reformatting noise. Its a result of running `mx eclipseformat` (a requirement to pass our gates)